webview/src/experiments/components/App.test.tsx

```diff
@@ -372,6 +377,213 @@ describe('App', () => {
+            type: MessageToWebviewType.SET_DATA
+          }
+        })
+      )
+
+      return 2
+    }
+
+    const toggleExpansion = (row: HTMLElement) => {
+      const expandButton = within(row).getByTitle('Contract Row')
+
+      fireEvent.click(expandButton)
+    }
+
+    it('should be hidden when the parent row is expanded', () => {
+      renderWithAllRowsExpanded()
+      const row = getMiddleStatesTestRow()
+      const indicators = getCountIndicators(row)
+      expect(indicators).toHaveLength(0)
+    })
+
+    describe('Checkbox selection counter', () => {
+      it('should not be visible if no sub-row was checked', () => {
+        renderWithAllRowsExpanded()
+        const row = getMiddleStatesTestRow()
+        const indicator = getCheckboxCountIndicator(row)
+        expect(indicator).not.toBeInTheDocument()
+      })
+
+      it('should display the correct number of checked sub-rows when the parent is collapsed', () => {
+        renderWithAllRowsExpanded()
+        const row = getMiddleStatesTestRow()
+        const numberOfSubrowsSelected = selectSomeSubRows()
+        expect(getCheckboxCountIndicator(row)).not.toBeInTheDocument()
+        toggleExpansion(row)
+        const collapsed = getMiddleStatesTestRow()
+        expect(getCheckboxCountIndicator(collapsed)).toHaveTextContent(
+          `${numberOfSubrowsSelected}`
+        )
+      })
+    })
+
+    describe('Stars counter', () => {
+      it('should not be visible if no sub-row was starred', () => {
+        renderWithAllRowsExpanded()
+        const row = getMiddleStatesTestRow()
+        const indicator = getCountIndicatorById(row, 'row-action-star')
+        expect(indicator).not.toBeInTheDocument()
+      })
+
+      it('should display the correct number of starred sub-rows when the parent is collapsed', () => {
+        renderWithAllRowsExpanded()
+        const row = getMiddleStatesTestRow()
+        const numberOfSubrowsStarred = starSomeSubRows()
+        expect(
+          getCountIndicatorById(row, 'row-action-star')
+        ).not.toBeInTheDocument()
+        toggleExpansion(row)
+        const collapsed = getMiddleStatesTestRow()
+        expect(
+          getCountIndicatorById(collapsed, 'row-action-star')
+        ).toHaveTextContent(`${numberOfSubrowsStarred}`)
+      })
+    })
+
+    describe('Runs counter', () => {
+      it('should not be visible if no sub-row is running', () => {
+        renderWithAllRowsExpanded()
+        const row = getMiddleStatesTestRow()
+        const indicator = getCountIndicatorById(row, 'row-subrows-running')
+        expect(indicator).not.toBeInTheDocument()
+      })
+
+      it('should display the correct number of running sub-rows when the parent is collapsed', () => {
+        renderWithAllRowsExpanded()
+        const row = getMiddleStatesTestRow()
+        const numberOfSubrowsRunning = runSomeExperiments()
+        expect(
+          getCountIndicatorById(row, 'row-subrows-running')
```

Comment:
[I] Seeing as we are no longer showing this information for the HEAD commit we no longer need the sub row running indicator. We should remove all of the code and shift the chevron, etc in the first cell to the left (saves space).